input/fsh/DEF_Cap_mcode-receiver-patients-in-group.fsh

```diff
@@ -4,8 +4,8 @@ Usage: #definition
an></td><!-- Supported _includes --><td></td><!-- Supported _revincludes --><td><!----></td><!-- Supported Operations --><td></td></tr><tr><td><a href=\"#group\"><span style=\"white-space: nowrap;\">Group<span title=\"SHALL\"><sup>&#x2666;</sup></span></span></a></td><td><span style=\"white-space: nowrap;\">search-type<span title=\"SHALL\"><sup>&#x2666;</sup></span></span></td><!-- Supported Profiles --><td></td><!-- Supported Searches will need to do some sort of mapping instead of join to get the expectations printed out --><td><span style=\"white-space: nowrap;\">code<span title=\"SHALL\"><sup>&#x2666;</sup></span></span></td><!-- Supported _includes --><td></td><!-- Supported _revincludes --><td><!----></td><!-- Supported Operations --><td></td></tr></tbody></table><!-- Resource Details --><h4 class=\"no_toc\" id=\"patient\">Patient</h4><p>Conformance Expectation:<strong>SHALL</strong></p><p>Supported Profiles:</p><ul><li><strong>SHALL</strong> support:<a href=\"http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-cancer-patient\">Cancer Patient Profile</a></li></ul><p>Profile Interaction Summary:</p><ul><li><strong>SHALL</strong> support <code>read</code>.</li></ul><p>Fetch and Search Criteria:</p><ul><li>A Client <strong>SHALL</strong> be capable of fetching a Patient resource using:<code class=\"highlighter-rouge\">GET [base]/Patient/[id]</code></li></ul><!-- Only included search parameters that are not MAY or there is no conformance requirement included --><p>Search Parameter Summary:</p><table class=\"grid\"><thead><tr><th>Conformance</th><th>Parameter</th><th>Type</th><th>Example</th></tr></thead><tbody><tr><td><strong>SHALL</strong></td><td><a href=\"http://hl7.org/fhir/us/core/STU5.0.1/SearchParameter-us-core-patient-id.html#id\">_id</a></td><td>token</td><td><code class=\"highlighter-rouge\">GET [base]/Patient?_id=[id]</code></td></tr></tbody></table><h4 class=\"no_toc\" id=\"group\">Group</h4><p>Conformance Expectation:<strong>SHALL</strong></p><p>Profile Interaction Summary:</p><ul><li><strong>SHALL</strong> support <code>search-type</code><sup>&#8224;</sup>.</li></ul><blockquote><p>search-type<sup>&#8224;</sup></p><p>Retrieve Group of references to Patient resources conforming to mCODE.</p></blockquote><p>Fetch and Search Criteria:</p><ul><li>A Client <strong>SHALL</strong> be capable of fetching resources matching a search query using:<code class=\"highlighter-rouge\">GET [base]/Group/[id]{?[parameters]{&amp;_format=[mime-type]}}</code></li></ul><!-- Only included search parameters that are not MAY or there is no conformance requirement included --><p>Search Parameter Summary:</p><table class=\"grid\"><thead><tr><th>Conformance</th><th>Parameter</th><th>Type</th><th>Example</th></tr></thead><tbody><tr><td><strong>SHALL</strong></td><td><a href=\"http://hl7.org/fhir/R4/group.html#search\">code</a></td><td>token</td><td><code class=\"highlighter-rouge\">GET [base]/Group?code=[system]|[code]</code></td></tr></tbody></table></div>"
```

Comment:
I suggest deleting these text.divs  (in all Cap Statmts) for good, rather than commenting.  
Even if you did want to revive them, you'd. need to regenerate.